extension/_locales/en/messages.json

```diff
@@ -919,6 +919,10 @@
     "message": "Show Number of Cookies for that Domain over the Icon",
     "description": "Show Number of Cookies for that Domain over the Icon"
   },
+  "siteDataText": {
+    "message": "Site Data",
+    "description": "Site Data.  Found in Activity Log Summary and Notifications."
```

Comment:
Looks like there's an extra space between `Data.  Found`

Reply:
It's only a description.  During my high school computing classes, I had been instructed to use double-spacing after each period, despite never used a typewriter before.  I haven't given it much thought about changing it back to one space (including after colons), so this habit persisted over the last 10 years.  Of course this only happens with a physical keyboard.  Software keyboards can auto correct my two spaces back into one.